zen-quotes-app/src/app/authors/authors.component.ts

```diff
@@ -0,0 +1,25 @@
+import {Component, EventEmitter, Input, Output} from '@angular/core';
+import {Author} from "../../types/author.type";
+import { faChevronDown } from '@fortawesome/free-solid-svg-icons';
+import {CheckboxControlValueAccessor} from "@angular/forms";
+
+@Component({
+  selector: 'app-authors',
+  templateUrl: './authors.component.html',
+  styleUrls: ['./authors.component.scss']
+})
+
+export class AuthorsComponent {
+  faChevronDown = faChevronDown;
+  authorsQuantity: number = 23;
+  authorsFilter: string[] = [];
+
+  @Input() authors: string[] = [];
+  @Output() addValueToFilter = new EventEmitter();
+
+  showMoreAuthors() {
```

Comment:
Challenge: can you add unit tests that verify whether this function works as expected?

Thanks! 🙏 🙇